### PR TITLE
test(module:anchor): added missing tests

### PR DESCRIPTION
**Pull request by ParsaArvanehPA**
## PR Checklist
Please check if your PR fulfills the following requirements:

- [✔] The commit message follows our guidelines: https://github.com/NG-ZORRO/ng-zorro-antd/blob/master/CONTRIBUTING.md#commit
- [✔] Tests for the changes have been added (for bug fixes / features)
- [ ] Docs have been added / updated (for bug fixes / features)

## PR Type
What kind of change does this PR introduce?

- [ ] Bugfix
- [ ] Feature
- [ ] Code style update (formatting, local variables)
- [ ] Refactoring (no functional changes, no api changes)
- [ ] Build related changes
- [ ] CI related changes
- [ ] Documentation content changes
- [ ] Application (the showcase website) / infrastructure changes
- [ ] Other... Please describe: missing unit tests for the Anchor Component

## What is the new behavior?
|               | Before  | After  |
|---------------|---------|--------|
| **Statements**| 99.3%  | 100%   |
| **Branches**  | 89.79%   | 100%   |
| **Functions** | 100%  | 100%   |
| **Lines**     | 99.27%  | 100%   |

## Does this PR introduce a breaking change?
- [ ] Yes
- [✔] No

### CODE REVIEW

**Comment by ParsaArvanehPA** [comment]
Hi guys
@Laffery I think you meant to close and merge, instead you clicked close in this [PR](https://github.com/NG-ZORRO/ng-zorro-antd/pull/8733).

**Comment by Laffery** [comment]
> Hi guys @Laffery I think you meant to close and merge, instead you clicked close in this [PR](https://github.com/NG-ZORRO/ng-zorro-antd/pull/8733).

I dont know, I have no operations on this PR.

Please fix the CI issues first:)

**Review by Laffery — APPROVED** [review]
LGTM

**Comment by ParsaArvanehPA** [comment]
> > Hi guys @Laffery I think you meant to close and merge, instead you clicked close in this [PR](https://github.com/NG-ZORRO/ng-zorro-antd/pull/8733).
> 
> I dont know, I have no operations on this PR.
> 
> Please fix the CI issues first:)

The failed test is for the Image component, I made this [Pr](https://github.com/NG-ZORRO/ng-zorro-antd/pull/8766) for it.
It seems to have fixed the problem... .

**Comment by Laffery** [comment]
> > > Hi guys @Laffery I think you meant to close and merge, instead you clicked close in this [PR](https://github.com/NG-ZORRO/ng-zorro-antd/pull/8733).
> > 
> > 
> > I dont know, I have no operations on this PR.
> > Please fix the CI issues first:)
> 
> The failed test is for the Image component, so I made this [Pr](https://github.com/NG-ZORRO/ng-zorro-antd/pull/8766) for it. It seems to have fixed the problem... .

I had merged the fix PR for image compnent, please rebase this pr and trigger the CI againe

**Comment by Laffery** [comment]
It seems that the updating for `tick` duration from 300 to 600 still not works :)
What about increasing it to 1000?

**Comment by ParsaArvanehPA** [comment]
> @ParsaArvanehPA It seems that the updating for `tick` duration from 300 to 600 in image.spec.ts still not works :) What about increasing it to 1000?

This behavior is kinda random. some of today's pr are fixed and some aren't.
Yeah let's get it up to 1000ms.
I will first test it in this branch, if it works I will then create a new PR for it.

**Comment by HyperLife1119** [comment]
How about converting PLACEHOLDER to Data URL?

```
const PLACEHOLDER = `data:image/png;base64,...`
```

**Comment by ParsaArvanehPA** [comment]
> How about converting PLACEHOLDER to Data URL?
> 
> ```ts
> const PLACEHOLDER = `data:image/png;base64,...`
> ```

Could work 👍
made the change, waiting for the result

**Review by Laffery — APPROVED** [review]
LGTM

File: components/image/image.spec.ts [hunk 1]
```diff
@@ -137,7 +137,7 @@ describe('Placeholder', () => {
     context.placeholder = PLACEHOLDER;
     const image = debugElement.nativeElement.querySelector('img');
     fixture.detectChanges();
-    tick(600);
+    tick(1000);
```
Comment (HyperLife1119) [inline components/image/image.spec.ts:140]: If DataURL is used, is `tick(1000)` still necessary?

Comment (ParsaArvanehPA) [reply components/image/image.spec.ts:140]: No I don't thinks so, should i revert it in another PR, since this one is merged?

Comment (HyperLife1119) [reply components/image/image.spec.ts:140]: Yes of course.

Comment (ParsaArvanehPA) [reply components/image/image.spec.ts:140]: Apparently it is needed.
https://dev.azure.com/ng-zorro/NG-ZORRO/_build/results?buildId=7095&view=ms.vss-test-web.build-test-results-tab&runId=21460&resultId=100799&paneView=debug

**Pull request merged**